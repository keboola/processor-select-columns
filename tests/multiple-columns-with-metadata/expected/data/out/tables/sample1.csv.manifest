{"delimiter":",","enclosure":"\"","columns":["col3","col2"],"column_metadata":{"col2":[{"key":"KBC.datatype.nullable","value":true}],"col3":[{"key":"KBC.datatype.nullable","value":true}]}}
