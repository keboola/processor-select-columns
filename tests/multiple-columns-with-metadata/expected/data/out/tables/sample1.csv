"val3","val2"
"val3","val4"
